app/templates/components/about-alpha.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="about-alpha__container">
+  {{#each videos as |video|}}
+    <a class="about-alpha__video" href="https://youtu.be/{{video.videoId}}" target="_blank" title="{{video.title}}">
```

Comment:
Could you use the YoutubeVideo component? That does most of this things for you and does some smart things with the thumbnail

Reply:
You don't want to play the video in an iframe when it is that small